hcipy/optics/fiber.py

```diff
@@ -3,7 +3,35 @@
 from .optical_element import OpticalElement, AgnosticOpticalElement, make_agnostic_forward, make_agnostic_backward
 from .wavefront import Wavefront
 from ..mode_basis import ModeBasis, make_lp_modes
-from ..field import Field
+from ..field import Field, CartesianGrid, RegularCoords
+from ..dev import deprecated
+
+def make_gaussian_fiber_mode(mode_field_diameter):
+	'''The Gaussian approximation of a fiber mode.
+
+	Parameters
+	----------
+	mode_field_diameter : scalar
+		The mode field diameter of the fiber.
+
+	Returns
+	-------
+	Field generator
+		The Gaussian fiber mode.
+	'''
+	def func(grid):
+		if grid.is_('cartesian'):
+			r2 = grid.x**2 + grid.y**2
+		else:
+			r2 = grid.as_('polar').r**2
+
+		# Compute and normalize the Gaussian function.
+		res = np.exp(-r2 / ((0.5 * mode_field_diameter)**2))
+		res /= np.sum(np.abs(res)**2 * grid.weights)
```

Comment:
We know theoretically what the normalization factor should be, right? It's 1/sqrt(2*pi * sigma^2) with some correction for the pixel size. Wouldn't that be a better normalization? I ran into issues before where I couldn't rely on numerical normalization due to a finite grid size.

Reply:
Fixed. There is no correction for pixel size. However, there is a change inside the exponential itself, since we are normalizing for the square, not the Gaussian itself.

BTW, should we then also remove the normalization inside the `SingleModeFiberInjection` element itself? That feels wrong though.

Reply:
Yeah I think we need to leave the normalization inside that class just in case some weird mode is used there.

Reply:
The normalisation is important, otherwise, the throughput will look oddly too high. Also, the mode_field_diameter requires a clear definition. It can change in textbooks. Usually, I use the same as Thorlabs: 1/e2 on the power (https://www.thorlabs.com/NewGroupPage9_PF.cfm?ObjectGroup_ID=949). So it would be:

		res = np.exp(-r2 / ((mode_field_diameter)**2))
		res *= np.sqrt(2/np.pi)/mode_field_diameter
		
(from heart, but please double check!)

Reply:
I kept the numerical normalization inside the `SingleModeFiberInjection` class.

@sylacour Thanks. I indeed had a wrong definition of the MFD. (I intended to use the 1/e2 one, but I made a mistake in evaluating the Gaussian.) Your from-heart formula was actually correct, except for exchanging MFD with the mode field radius. I also added a test to make sure that the 1/e definition is always checked for this function. The normalization was already tested.